packages/cli/src/commands/trace.ts

```diff
@@ -66,7 +65,7 @@ const commandModule: CommandModule<Options, Options> = {
 
     // Create World contract instance from deployed address
     const provider = new ethers.providers.StaticJsonRpcProvider(rpc);
-    const WorldContract = new ethers.Contract(worldAddress, IBaseWorldData.abi, provider) as IBaseWorld;
```

Comment:
correct! this would have been the last remaining spot to use typechain, so I opted to ditch it all for the short time that we're between ethers and viem for deploys